test/playback.test.js

```diff
@@ -287,29 +287,33 @@ QUnit[testFn]('DASH sidx with alt audio should end', function(assert) {
   });
 });
 
-QUnit[testFn]('loops', function(assert) {
-  const done = assert.async();
-  const player = this.player;
+// TODO: why does this make the next test
+// throw an "The operation was aborted." on firefox
+if (!videojs.browser.IS_FIREFOX) {
```

Comment:
Firefox 69 started throwing an error "The Operation was aborted", and no amount of promise silencing seems to help. The test itself actually passes, but it throws an error that causes a global failure.

Reply:
This happens in master too.

Reply:
I wonder if this should instead use `QUnit.skip`? Maybe `QUnit[videojs.browser.IS_FIREFOX ? 'skip' : testFn]`?